plugins/catalog-backend/api-report.md

```diff
@@ -867,14 +867,15 @@ export type EntityAncestryResponse = {
   }>;
 };
 
+// Warning: (ae-forgotten-export) The symbol "AndEntityFilter" needs to be exported by the entry point index.d.ts
+// Warning: (ae-forgotten-export) The symbol "OrEntityFilter" needs to be exported by the entry point index.d.ts
```

Comment:
Note these.

I see what you did with the `is` checks etc, and it looks good,  but these extra types become part of the package's exported API surface, bloating it a little.

You could make them public and documented etc as well, but I'm not sure it's worth it - probably better to just put them literally in the code:

```ts
export type EntityFilter =
  | { allOf: EntityFilter[] }
  | { anyOf: EntityFilter[] }
  | EntitiesSearchFilter;
```

and deal with that explicitly without dedicated type names.

Reply:
Done!